app/src/main/resources/stream-registry.graphql

```diff
@@ -0,0 +1,102 @@
+type KeyValue {
```

Comment:
I had to copy this file from another module. 

Try to build & run the container without this here, and it says cannot find `**/*.graphql` files on the classpath.